index.html

```diff
@@ -13,219 +13,270 @@
     <link rel="stylesheet" href="index.css" />
   </head>
   <body>
-    <header class="app-bar">
-      <a href="/" class="logo"
-        ><img src="images/logo.svg" alt="Home" width="76" height="20"
-      /></a>
-      <nav>
-        <a href="#menu" class="tertiary-button"
-          ><img src="images/icons/menu.svg" alt="Menu" width="24" height="24"
+    <div class="headline-container">
+      <header class="app-bar">
+        <a href="/" class="logo"
+          ><img src="images/logo.svg" alt="Home" width="76" height="20"
```

Comment:
Nice job with the navbar :+1: 

**[ OPTIONAL ]**
- I believe your portfolio will look great if you can make the navbar stick to the top of the page. This is a very good practice for the user experience of your portfolio. Making the navbar stick to the top of the page will give the users access to your navbar no matter which section they are in your portfolio. It also enhances easy navigation from one section to another. Therefore, I think you should give it a shot :smile: 
- Another optional feature is to change the `cursor` behavior of your buttons to `pointer` on `hover`. When users move the mouse cursor over your buttons, it looks nice and better if the cursor can change to pointer.

Reply:
Well done on fixing what the previous reviewer suggested even if it was optional. Good job!